uw_spotseeker/__init__.py

```diff
@@ -35,52 +34,40 @@ class Spotseeker(object):
     def post_image(self, spot_id, image: bytes) -> bytes:
         url = "/api/v1/spot/%s/image" % spot_id
 
-        headers = {"X-OAuth-User": settings.OAUTH_USER}
+        headers = {"X-OAuth-User": os.getenv('OAUTH_USER')}
         headers['files'] = {'image': ('image.jpg', image)}
 
-        try:
-            response = Spotseeker_DAO().postURL(url, headers)
-            status = response.status
-            if status not in [200, 201]:
-                raise DataFailureException(url,
-                                           status,
-                                           getattr(response, 'reason', None))
-        except AttributeError:
-            raise NotConfigured("must set credential in settings")
+        response = Spotseeker_DAO().postURL(url, headers)
+        status = response.status
+        if status not in [200, 201]:
+            raise DataFailureException(url, status,
+                                       getattr(response, 'reason', None))
 
         return response.data
 
     def delete_image(self, spot_id, image_id, etag) -> None:
         url = "/api/v1/spot/%s/image/%s" % (spot_id, image_id)
 
-        try:
-            headers = {"X-OAuth-User": settings.OAUTH_USER,
-                       "If-Match": etag}
-            response = Spotseeker_DAO().deleteURL(url, headers)
-            content = response.data
-            status = response.status
-
-        except AttributeError:
-            raise NotConfigured("Must set OAUTH_USER in settings")
```

Comment:
removed these try-except blocks because os.getenv() will return None and not error